roles/edpm_neutron_dhcp/templates/wrappers/dnsmasq.j2

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+{% if edpm_neutron_dhcp_sidecar_debug -%}
```

Comment:
@booxter The guidance we've received is that if an individual file we're copying (or copying non-trivial amounts of content from) contains a copyright/attribution notice, we should retain that in our new file. Additionally, if the source repository's README contains a global notice of copyright/attribution, we should retain that in our README.

If, on the other hand, neither of those things are true (which I believe is the case with at least this particular file), then we do not need to reconstruct a copyright/attribution listing from the source repo's history (which is good because there's like 160 committers to puppet-tripleo!)